extensions/default/src/index.js

```diff
@@ -2,9 +2,9 @@ import getDataSourcesModule from './getDataSourcesModule.js';
 import getLayoutTemplateModule from './getLayoutTemplateModule.js';
 import getPanelModule from './getPanelModule';
 import getSopClassHandlerModule from './getSopClassHandlerModule.js';
-import getHangingProtocolModule from './getHangingProtocolModule.js';
```

Comment:
Is there a good reason for this change here?  It seems an empty change...

Reply:
That was a mistake